### PR TITLE
Update Meta IO

### PR DESCRIPTION
**Pull request by dzenanz**

### CODE REVIEW

**Comment by seanm** [comment]
Looks like only my changes?  That doesn't really make me an ideal reviewer. :)

**Comment by dzenanz** [comment]
@bradking I don't quite understand why [ghostflow fails](https://github.com/InsightSoftwareConsortium/ITK/pull/1285/checks?check_run_id=244907468). The commit doesn't touch the file in question.

**Comment by bradking** [comment]
@dzenanz the ghostflow bot's complaint is

```
commit b3e50e5 adds bad whitespace:

    src/doc/README.txt:4: new blank line at EOF.
```

One can run `git show b3e50e5:src/doc/README.txt` to see that the file does have an extra blank line at the end.  It's complaining about the path as it exists in MetaIO, not after merging to ITK.

**Comment by dzenanz** [comment]
https://github.com/Kitware/MetaIO/pull/84 removes the white space. Let's merge that so I can update this PR.

File: Modules/ThirdParty/MetaIO/src/MetaIO/src/metaCommand.cxx [hunk 1]
```diff
@@ -1722,8 +1722,8 @@ bool MetaCommand::Parse(int argc, char* argv[])
   std::string tag = "";
   std::string args;
 
-  unsigned int currentField = 0; // current field position
-  int currentOption = 0; // id of the option to fill
+  unsigned long currentField = 0; // current field position
```
Comment (issakomi) [inline Modules/ThirdParty/MetaIO/src/MetaIO/src/metaCommand.cxx]: May be _long long_, otherwise there is no effect on Windows.

Comment (seanm) [reply Modules/ThirdParty/MetaIO/src/MetaIO/src/metaCommand.cxx]: I changed `currentOption` to long because `GetOptionId` returns `long`.

**Pull request merged**